### ISSUE TITLE
Edit Multi-Collection Portals

### ISSUE THREAD

**Issue by campmlc** [ArctosDB/arctos#1787]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Is your feature request related to a problem? Please describe.**
We need a way to edit/add to narrative text description for multi-collection portals

**Additional context**
Arkansas State (ACBC) needs to add narrative description for the ASUMZ and STAR multi collections portal on portals page:

The Arkansas State University Museum of Zoology (ASUMZ)

The Arkansas State University Museum of Zoology (ASUMZ) is an active group of research and teaching collections that joined with the herbarium in 2016 to unify as the Arkansas Center for Biodiversity Collections (ACBC) within the Department of Biological Sciences at Arkansas State University. ASUMZ includes collections of aquatic macroinvertebrates (mostly larval insects), bivalves, recent mammals, reptiles and amphibians, fishes, insects, and birds. There is a heavy regional focus within Arkansas, with specific geographic areas of interest being the Ozark Plateaus region, Crowley’s Ridge, and the Mississippi Alluvial Plain. Some of the ASUMZ collections represent the largest collections in Arkansas for the taxonomic group, including aquatic macroinvertebrates, bivalves, fishes, reptiles and amphibians, and mammals.

The Arkansas State University Herbarium (STAR)

The Arkansas State University Herbarium (STAR) is an active pair of research and teaching collections that joined with the ASUMZ in 2016 to unify as the Arkansas Center for Biodiversity Collections (ACBC) within the Department of Biological Sciences at Arkansas State University.  Recently, a newly created diatom collection from a sister campus in the Arkansas State University System joined STAR.  The vascular and non-vascular plant collection is the oldest natural history collection at Arkansas State University and leads the state in efforts of digitization, resource sharing, and data dissemination.

**Priority**
I would like to have this resolved by date: ___soon____

**Comment by acdoll** [comment]
Doesn't this involve submitting a revised .eml file to VertNet? Most of the DMNS descriptions on these aggregators also need updating. It would be great if there was a form on Arctos that we could use to make this happen (somewhere under Manage Collection, perhaps?). Or maybe a button under the collection description field that can be clicked to push changes to the VertNet IPT?

**Comment by dustymc** [comment]
I think IPT can't deal with files or services or something - I've offered to generate that a few times but no takers. There is a form to make copy-pasting easier.

<img width="281" alt="screen shot 2018-11-08 at 2 41 27 pm" src="https://user-images.githubusercontent.com/5720791/48232463-902f7a00-e366-11e8-821a-faf457b21ab4.png">

**Comment by Jegelewicz** [comment]
> Most of the DMNS descriptions on these aggregators also need updating. It would be great if there was a form on Arctos that we could use to make this happen (somewhere under Manage Collection, perhaps?).

All of UTEP eml files need to be updated too and this is something I requested a long time ago. We need to collect the data required by the ipt in Arctos and then have the ability to generate the eml file to send to VertNet. @acdoll I think it would help if we started with an eml mock-up where we put the fields currently in the "manage collection" form into the eml, then we see what is missing from Arctos "manage collection" and see if we can add those fields there to complete a "push to eml" function. Care to take that on?
